components/authz-service/README.md

```diff
@@ -617,6 +617,7 @@ and access to all _administrative_ tasks to all users in the local admins group
 (`team:local:admins`).
 This allows you to explore the user interface immediately,
 unencumbered with policy roadblocks limiting what you can see.
+<!-- TODO: link is broken any reference? -->
 The set of default policies is [here](../../docs/default_policies.md).
```

Comment:
I'd suggest removing the TODO and linking to the current documentation for the default policies.

```suggestion
The set of default policies is described [here](../../components/automate-chef-io/content/docs/iam-v2-overview.md).
```

Reply:
Have added TODO comment to pointed it out. Thanks